app/metrics.yaml

```diff
@@ -18,28 +18,42 @@ events:
       startups, not just cold startup. Note: There is a short gap
       between the time application goes into background and the time
       android reports the application going into the background.
-      Note: This metric does not cover the following cases:
-      Case # 1 -> a). open a link(for example, gmail) with in-app
-      Browser (metric report custom_tab startup) b). press home button
-      c). open gmail again (which brings us back to in app browser).
-      Step c will not report startup metric. Case # 2 -> a). open fenix
-      b). press home button c). launch fenix through app switcher/recent
-      apps. step c will not report startup type.
+      Note: This metric does not record souce when app opened from
+      task switcher: open application -> press home button -> open
+      recent tasks -> choose fenix. In this case will report
+      [source = unknown, type = hot, has_saved_instance_state = false].
+      unknown is for tasks switcher or other unanticipated startup source.
+      Error is for impossible cases.
     extra_keys:
       source:
         description: |
           The method used to open Fenix. Possible values are `app_icon`,
-          `custom_tab`, `link` or `unknown`
+          `custom_tab`, `link`, `unknown` or `error`.
+      startup_type:
```

Comment:
nit: can we move this above `source`? I'd consider `type` to be the primary key using my intuition, particularly if we're storing the error state there. It's important for folks to see that ERROR state as early as possible.